lib/chromic_pdf/pdf/template.ex

```diff
@@ -0,0 +1,172 @@
+defmodule ChromicPDF.Template do
+  @moduledoc """
+  This module contains basic styles and options that make it simple to make the HTML content fully
+  cover a given page.
+  """
+
+  require EEx
+
+  @type blob :: binary()
+
+  @type content_option ::
+          {:html, blob()}
```

Comment:
Why did you choose `:html` for this variable name?
I fear that it can lead to confusion. `:header` and `:footer` are valid HTML too.
What about `:body`, `:page_content`, or simply `:content`?

Reply:
🤔 I wanted to align it with [`ChromicPDF.print_to_pdf/2`](https://github.com/bitcrowd/chromic_pdf/blob/0f0064c5c84c3ff6b055c873ffd885f2a3ebfc5e/lib/chromic_pdf/supervisor.ex#L114) where I also called it `:html`.

But you're right, it is indeed confusing and might even be a bit misleading in this case, as the param should only be body content markup, and don't have the enclosing `<html></html>` tags... Chromium has this print template in which they call it `content`, so I might go with this (and change the `:html` in `print_to_pdf/2`, as well) https://cs.chromium.org/chromium/src/components/printing/resources/print_header_footer_template_page.html?g=0&l=125

Reply:
hmm realized the param is actually called `html` in devtools, so going to keep it htere. But can still change it in the template, because why not 🤷‍♂️

https://chromedevtools.github.io/devtools-protocol/tot/Page#method-setDocumentContent